site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,57 +16,157 @@ updated: 2021-08-04
+and enable up to 10 rulsets
+([MAX_NUMBER_OF_ENABLED_STATIC_RULESETS](/docs/extensions/reference/declarativeNetRequest/#property-MAX_NUMBER_OF_ENABLED_STATIC_RULESETS))
+at a time.
 
-The [policy update blog post](/blog/policy-update-2sv/) published on 2021.06.29 has been updated to
-correct the two step verification deployment timeline.
+### Chrome 93: Cross origin isolation support
+
+Both [Manifest V2](/docs/extensions/mv2/cross-origin-isolation/) and [Manifest
+V3](/docs/extensions/mv3/cross-origin-isolation/) extensions can now opt into [cross origin
+isolation](https://web.dev/cross-origin-isolation-guide/). This feature limits which cross-origin
+resources can load an extension's pages and enables the use of low level web platform features like
+[SharedArrayBuffer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer). Opt in will be required starting in Chrome 95.
 
-### Developer program policies updated
+### 2021-08-04: Two step verification enforcement {: #two-step-verification-enforcement }
 
-#### Published 2021.06.29
+The [policy update blog post](/blog/policy-update-2sv/) published on 2021-06.29 has been updated to
+correct the two step verification deployment timeline.
+
+### 2021-06-29: Developer program policies updated {: #developer-program-policies-updated }
 
 The Chrome Web Store [Developer Program Policies](/docs/webstore/program_policies) have been updated
 with clarifications to the deceptive installation tactics, spam, and repetitive content policies.
 This update also includes a new two step verification requirement to publish on the Chrome Web
 Store. [Read the blog post](/blog/policy-update-2sv/) for more information.
 
-### New blog post: "Extension actions in Manifest V3"
-
-#### Published 2021.06.23
+### 2021.06.23: "Extension actions in Manifest V3" blog post {: #new-blog-post-extension-actions-in-manifest-v3 }
 
-Chrome extensions had Browser and Page Actions APIs for years, but Manifest V3 replaced both with a
-generic [Actions API](/docs/extensions/reference/action/). This post explores the history of these
-APIs and what has changed in Manifest V3. [Read the post](/blog/mv3-actions).
+Chrome extensions had `chrome.browserAction` and `chrome.pageActions` APIs for years, but Manifest
+V3 replaced both with a generic [`chrome.actions`](/docs/extensions/reference/action/) API. This
+post explores the history of these APIs and what has changed in Manifest V3. [Read the
+post](/blog/mv3-actions).
 
-### New blog post: "Introducing chrome.scripting"
+### 2021.06.08: "Introducing chrome.scripting" blog post {: #new-blog-post-introducing-chromescripting }
 
-#### Published 2021.06.08
+The [`chrome.scripting`](/docs/extensions/reference/scripting/) API is a new Manifest V3 API focused
+on, well, scripting. In this post we dig into the motivations for this change and take a closer look
+at some of the new capabilities it introduces. [Read the post](/blog/crx-scripting-api).
```

Comment:
>at some of the new capabilities it introduces

This could be tighter:

- at it's new capabilities.

or 

- at capabilities it introduces.

Reply:
Ack. Using "at it's new capabilities".